### PR TITLE
Gulp task for generating the classmap

### PR DESCRIPTION
**Pull request by charleswaddell**

### CODE REVIEW

**Comment by charleswaddell** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;aK934l2h&#x2F;995-0-5d-gulp-task-for-rebuilding-classmap-on-file-changes"> 0.5d - Gulp task for rebuilding classmap on file changes</a>

File: lib/phpclassmap.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+var execFile = require('child_process').execFile;
+
```
Comment (gauthierm) [inline lib/phpclassmap.js]: blank line not needed

File: lib/phpclassmap.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+module.exports = {
+	task: function() {
+		var callback = function (error, stdout, stderr) {
+			if (error !== null) {
+				console.log('exec error: ' + error);
+			}
+		};
+
+		execFile( 'composer', [ '-q', 'dump-autoload' ], callback);
```
Comment (gauthierm) [inline lib/phpclassmap.js]: no space needed after opening `(`

File: lib/phpclassmap.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+var execFile = require('child_process').execFile;
+
+var paths = require('./paths');
+
+module.exports = {
+	task: function() {
+		var callback = function (error, stdout, stderr) {
+			if (error !== null) {
+				console.log('exec error: ' + error);
```
Comment (gauthierm) [inline lib/phpclassmap.js]: Should not use `console.log`. Use `gutil.log`

**Pull request merged**